setup.py

```diff
@@ -77,6 +77,7 @@ def prerelease_local_scheme(version):
         'Pillow!=8.3.0,!=8.3.1',
         'psutil>=4.2.0',  # technically optional
         'numpy>=1.10.4',
+        'importlib-metadata ; python_version < "3.8"',
```

Comment:
@manthey, on the next release, I need to make sure this is captured in the conda recipe for py3.7

Reply:
Mostly a reminder for myself, and hoping you'll remind me if I forget